lib/susedistribution.pm

```diff
@@ -198,6 +198,7 @@ sub become_root {
     my ($self) = @_;
 
     $self->script_sudo('bash', 0);
+    wait_still_screen;
```

Comment:
we don't wan't to wait in type_string so have add this everywhere where it fails